escapades_backend/spec/requests/current_user_spec.rb

```diff
@@ -7,7 +7,6 @@
   it 'current_user returns correct user' do
     auth_token = Devise::JWT::TestHelpers.auth_headers({}, user)['Authorization']
     get "/current_user", headers: { 'Authorization' => auth_token }
-    expect(user).to have_attributes(JSON.parse(response.body))
+    expect(user).to have_attributes(JSON.parse(response.body).reject {|k| k == "avatar_url"})
```

Comment:
Why?

Reply:
just to make that work 
see my comment : 
![image](https://github.com/Batch-440/escapades/assets/66632538/a18f71e7-00a2-45a5-9952-6a5f80ceaa8b)
I will try to do so

Reply:
Can you try to serialize the user in the expect part? So you can compare it with the avatar_url attribute

Reply:
ok that will do